.github/workflows/add-release-files.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  release:
+    types: [created, published, edited]
+name: Handle Release
+jobs:
+  generate:
+    name: Create release-artifacts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the web-components branch
+        uses: actions/checkout@v2
+        with:
+          ref: wc-automate-release-files
+      - uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - run: yarn
+      - run: yarn build
+      - run: yarn build-bindings
+      - run: yarn pack
+      - name: Upload the artifacts
+        uses: skx/github-action-publish-binaries@release-0.15
```

Comment:
`release-0.15` is the latest. I didn't want to use master like the [sample](https://github.com/marketplace/actions/github-action-publish-binaries#sample-configuration) suggests because some new push could break our workflow.